src/components/engineering/hackathons/Header.jsx

```diff
@@ -0,0 +1,9 @@
+const Citrushack = () => {
```

Comment:
component naming :sob:

Reply:
bro knows i copied and pasted it now haha